docs/sdk/migrations/6.0.0.md

```diff
@@ -0,0 +1,491 @@
+---
+sidebar_position: 3
+---
+
+import FeedbackComponent from "@site/src/pages/feedback.md";
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Migrating from 5.x.x to 6.x.x
+
+## Breaking Changes & Important Updates
+
+The most significant change in v6 is the combination of `authSig` and `sessionSigs`. In most functions, the client-side generated `authSig` will no longer be accepted as an argument, as seen in commonly used functions like `executeJs` and `pkpSign`. Instead, it will be used to generate `sessionSigs` to authenticate with the Lit nodes.
```

Comment:
We need to add that `authSig` is still being used in access control conditions, and it's required for decryption